apps/package.json

```diff
@@ -197,6 +197,10 @@
     "filesaver.js": "0.2.0",
     "jszip": "3.0.0",
     "query-string": "4.1.0",
-    "react-hot-loader": "^1.3.1"
-  }
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-hot-loader": "^1.3.1",
+    "reactjs-mappletooltip": "^1.4.63"
```

Comment:
Did this tooltip library give us things that the other one didn't? One thing I'd worry about is that it doesn't look as well supported (i.e. only 9 stars on github).

Also, we generally add our packages as dev-dependencies rather than dependencies (we already have devDependencies for react/react-dom for example)